src/solve.jl

```diff
@@ -26,6 +26,22 @@ end
             (a[1,1]*a[2,2] - a[1,2]*a[2,1])*b[3]) / d )
 end
 
+for Sa in 2:3  # not needed for Sa = 1;
+    @eval begin
+    @inline function solve(::Size{($Sa,$Sa)}, ::Size{Sb}, a::StaticMatrix{<:Any, <:Any, Ta}, b::StaticMatrix{<:Any, <:Any, Tb}) where {Sb, Ta, Tb}
+        d = det(a)
+        T = typeof((one(Ta)*zero(Tb) + one(Ta)*zero(Tb))/d)
+        c = similar(b, T)
+        for col = 1:Sb[2]
```

Comment:
It's possible that using `Base.OneTo(Sb[2])` instead of `1:Sb[2]` will be faster. Could you check?

Reply:
No difference. See last comment.

Reply:
OK, thanks for trying.